extendedset/src/main/java/io/druid/extendedset/intset/ConciseSetUtils.java

```diff
@@ -1,3 +1,21 @@
+/*
```

Comment:
This header isn't accurate, see discussion on https://github.com/apache/incubator-druid/issues/5835#issuecomment-404257393. I think the policy on https://www.apache.org/legal/src-headers.html#3party guides us regarding what to do here.

The CONCISE libraries definitely sound like third-party since they are "work submitted directly to the ASF for which the submitter is not the copyright owner or owner's agent". So, we're not supposed to modify the original license headers. Checking out https://github.com/metamx/extendedset/commit/02366f18385b3e082866e5bbbd38357ceee13d79#diff-33bd87cac0991642db64c5259e881ae0R41, this file was originally copied from another file ConciseSet.java, which had the following license header. I think we should start from this one.

```
 /* 
  * (c) 2010 Alessandro Colantonio
  * <mailto:colanton@mat.uniroma3.it>
  * <http://ricerca.mat.uniroma3.it/users/colanton>
  *  
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

But also, the Apache license says "You must cause any modified files to carry prominent notices stating that You changed the files". So I think we should modify the header to add that parts of it are also licensed to ASF (i.e. the parts that were changed after Alessandro wrote the original version). Maybe something like this will get the point across.

```
 /* 
  * (c) 2010 Alessandro Colantonio
  * <mailto:colanton@mat.uniroma3.it>
  * <http://ricerca.mat.uniroma3.it/users/colanton>
  *
  * Modified at the Apache Software Foundation (ASF).
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  * 
  *     http://www.apache.org/licenses/LICENSE-2.0
  * 
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```